neoload/commands/fastfail.py

```diff
@@ -0,0 +1,155 @@
+import click
+
+from commands import test_settings, test_results
+from neoload_cli_lib import user_data, tools, displayer, running_tools
+from neoload_cli_lib import tools
+import logging
+import yaml
+from datetime import datetime, date
+import time
+import sys
+import traceback
+from dateutil.relativedelta import relativedelta
+
+@click.command()
+@click.argument('command', type=click.Choice(['slas'], case_sensitive=False))
+@click.argument("name", type=str, required=False)
+@click.option("--stop", 'stop', is_flag=True, default=True, help="Doesn't wait the end of test")
+@click.option("--force", 'force', is_flag=True, default=True, help="Doesn't wait the end of test")
+@click.option("--max-failure", 'max_failure', type=int, default=0, help="Max SLA failure threshold; default is zero")
+def cli(command, name, stop, force, max_failure): #, max_occurs):
+    """Fails if certain conditions are met, such as per-run SLAs failed % of time"""
+    if not command:
+        tools.system_exit({'message': "command is mandatory. Please see neoload fastfail --help", 'code': 2})
+        return
+
+    if max_failure < 0 or max_failure > 100:
+        tools.system_exit({'message': "--max-failure percentage tolerance must be between 0 and 100", 'code': 2})
+        return
+
+
+    if command == "slas":
+        monitor_loop(name, stop, force, max_failure)
+
+    else:
+        tools.system_exit({'message': "Invalid command. Please see neoload fastfail --help", 'code': 2})
+
+def monitor_loop(name, stop, force, max_failure):
+    dt_started = datetime.now()
+    print('fastfail started: ' + str(dt_started))
+    dt_current = dt_started
+
+    if sys.stdin.isatty():
+        sys.stdout = Unbuffered(sys.stdout)
+
+    is_initializing = False
+    is_running = False
+    has_exited = False
+    fails = []
```

Comment:
this variable "fails" is never used here. This line can be remove

Reply:
Resolved with 37a6a7e